externs/sourcebuffer.js

```diff
@@ -11,5 +11,9 @@
  * @externs
  */
 
+
 /** @type {string} */
 SourceBuffer.prototype.mode;
+
+// TODO: need to figure out how to correctly override this
```

Comment:
Why does this need to be overridden?  It appears to be in the compiler's built-ins already: https://github.com/google/closure-compiler/blob/a4329316423d8478452e6330c6b887dd97ddef00/externs/browser/mediasource.js#L155